### PR TITLE
Add generic way to toggle data.all features

### PR DESCRIPTION
**Pull request by blitzmohit**
### Feature or Bugfix
- Feature

### Detail
- Ability to turn disable features via a single configuration
- Define feature boundaries for 
  - AWS token/console actions on both dataset & env
  - File uploads/actions

### Relates
- #473 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -10,10 +10,13 @@
             "active": true
         },
         "datasets": {
-            "active": true
-        },
-        "worksheets": {
```
Comment (blitzmohit) [inline config.json:15]: > 2023-06-28 12:25:36 Application started for envname= `dkrcompose` DH_DOCKER_VERSION:None
2023-06-28 12:25:36 Process SpawnProcess-1:
2023-06-28 12:25:36 Traceback (most recent call last):
2023-06-28 12:25:36   File "/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
2023-06-28 12:25:36     self.run()
2023-06-28 12:25:36   File "/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
2023-06-28 12:25:36     self._target(*self._args, **self._kwargs)
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/subprocess.py", line 76, in subprocess_started
2023-06-28 12:25:36     target(sockets=sockets)
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 68, in run
2023-06-28 12:25:36     return asyncio.run(self.serve(sockets=sockets))
2023-06-28 12:25:36   File "/usr/lib64/python3.8/asyncio/runners.py", line 44, in run
2023-06-28 12:25:36     return loop.run_until_complete(main)
2023-06-28 12:25:36   File "/usr/lib64/python3.8/asyncio/base_events.py", line 616, in run_until_complete
2023-06-28 12:25:36     return future.result()
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/server.py", line 76, in serve
2023-06-28 12:25:36     config.load()
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 448, in load
2023-06-28 12:25:36     self.loaded_app = import_from_string(self.app)
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 24, in import_from_string
2023-06-28 12:25:36     raise exc from None
2023-06-28 12:25:36   File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 21, in import_from_string
2023-06-28 12:25:36     module = importlib.import_module(module_str)
2023-06-28 12:25:36   File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
2023-06-28 12:25:36     return _bootstrap._gcd_import(name[level:], package, level)
2023-06-28 12:25:36   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
2023-06-28 12:25:36   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
2023-06-28 12:25:36   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
2023-06-28 12:25:36   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
2023-06-28 12:25:36   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
2023-06-28 12:25:36   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
2023-06-28 12:25:36   File "/code/./cdkproxymain.py", line 24, in <module>
2023-06-28 12:25:36     load_modules(modes={ImportMode.CDK})
2023-06-28 12:25:36   File "/code/./dataall/modules/loader.py", line 81, in load_modules
2023-06-28 12:25:36     _check_loading_correct(in_config, to_load)
2023-06-28 12:25:36   File "/code/./dataall/modules/loader.py", line 234, in _check_loading_correct
2023-06-28 12:25:36     raise ImportError(f"The package {module} has been imported, but it doesn't contain ModuleInterface")
2023-06-28 12:25:36 ImportError: The package dataall.modules.worksheets.db has been imported, but it doesn't contain ModuleInterface

File: frontend/src/views/Tables/TablePreview.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Card, CircularProgress } from '@mui/material';
 import * as PropTypes from 'prop-types';
 import { DataGrid } from '@mui/x-data-grid';
 import { styled } from '@mui/styles';
-import previewTable from '../../api/DatasetTable/previewTable';
```
Comment (blitzmohit) [inline frontend/src/views/Tables/TablePreview.js:7]: The file being referenced does not exist

File: config.json (hunk 1 above)
Comment (nikpodsh) [reply config.json:15]: Hm, I couldn't reproduce the issue in the latest modularization-main. Can you please try to merge/rebase your branch an check if you still see the issue?

File: config.json [hunk 3]
```diff
@@ -10,10 +10,13 @@
...
-            "active": true
+            "active": true,
+            "file_uploads": true,
```
[the remark this answers is not in the document]
Comment (blitzmohit) [reply config.json:14]: sure, that sounds good. Will update it in the next commit.

File: config.json (hunk 1 above)
Comment (nikpodsh) [reply config.json:15]: Ok, let's merge it with inactive worksheets. I will address it in the dashboard PR

**Review by nikpodsh — APPROVED** [review]

File: config.json (hunk 1 above)
Comment (nikpodsh) [reply config.json:15]: The dashboard PR has been merged and I see that you have rebased to the fresh `modulatization-main`. This line should be reverted

File: frontend/src/views/Tables/TablePreview.js (hunk 2 above)
Comment (nikpodsh) [reply frontend/src/views/Tables/TablePreview.js:7]: This issue has also been addressed

**Comment by nikpodsh** [comment]
@blitzmohit, I haven't know that you have no access to merge.
The PR looks good for me!

**Comment by blitzmohit** [comment]
Thanks @nikpodsh for your guidance on getting this merged.

**Pull request merged**